style.css

```diff
@@ -0,0 +1,3273 @@
+        padding: 0px;
+        display: none;
+    }
+
+    .news-one img{
+        width: 200.6px;
+        height: 143px;
+    }
+    .news-one {
+        width: 205.6px;
+        display: none;
+    }
+    .news-two{
+        width: 205.6px;
+       
+    }
+    .news-two img{
+        width: 200.6px;
+        height: 143px;
+    }
+    .news img{
+        width: 200px;
+        height: 143.5px;
+    }
+    .news_tab{
+        display: flex;
+        flex-wrap: wrap;
+    }
+    /* .news div{
+        padding: 0px;
+        width: 227px;
+    } */
+    .news p{
+        width: 200px;
+        font-size: 80%;
+    }
+    .news h5{
+        padding-bottom: 5px;
+    }
+    
+    
+    /* end of .news */
+    
+    
+    
+    .arts{
+        display: grid;
+        grid-template-columns: 1fr 1fr ;
+        padding: 0px;
+        width: 100%;
+        gap: 5px;
+    }
+    .arts img{
+        width: 200px;
+        height: 143.5px;
+    }
+    .arts div{
+        padding: 0px;
+    }
+    .arts p{
+        width: 215px;
+        font-size: 80%;
+    }
+    .arts h5{
+        padding-bottom: 5px;
+    }
+    
+    .news-two2{
+        padding-right: 0%;
+    }
+    
+    .news-one1{
+        padding-left: 0%;
+    }
+    
+    
+    
+    /* end of news */
+    
+    
+    .third_line-break{
+        border-top: 1px double rgb(159, 156, 156) ;
+        padding: .04%;
+        border-bottom: 1px double rgb(162, 160, 160);
+    }
+    
+    
+    
+    
+    .footer_div h5{
+        padding-bottom: 5px;
+    }
+    
+    
+    .footer_list{
+        list-style-type: none;
+       
+    }
+    .footer_li{
+        text-decoration: none;
+    }
+    
+    .F-news a{
+        display: block;
+        text-decoration: none;
+        color: black;
+        text-align: left;
+    }
+    
+    .footer-flex{
+        display: block;
+        justify-content: space-between;
+    }
+    
+    
+    .css-1oyjjn8 svg{
+        width: 13px;
+        height: 13px;
+    
+    }
+    
+    .css-1iruc8t{
+        list-style: none;
+    
+    }
+    
+    .css-1iruc8t a{
+        text-decoration: none;
+        color: black;
+    
+    }
+    
+    .css-6td9kr{
+        list-style: none;
+    }
+    
+    .css-6td9kr a{
+        text-decoration: none;
+        color: black;
+    }
+    
+    .css-6td9kr{
+        border-top: 1px solid rgb(142, 139, 139);
+        margin-top:15px ;
+        padding-top: 10px;
+    }
+    .F-subscribe{
+        border-left: 1px solid rgb(172, 170, 170);
+        padding-left: 10px;
+        
+    }
+    
+    .footerlogo{
+        width: 184px;
+        height: 24px;
+        
+    }
+    
+
+    /* changes  */
+
+
+    .tab{
+        display: none;
+    }
+
+
+    .article_tab{
+        display: none;
+    }
+
+    .fitness_tab{
+        display: none;
+    }
+    .play_tab{
+        display: none;
+    }
+
+   
+    
+
+    .arts_tab{
+        display: none;
+    }
+
+    .news_tab div{
+        padding-right: 9px;
+    }
+}
```

Comment:
fix the same type of issues for mobile and tab version that were in the desktop version. In general, make ur styling efficient by getting rid of unnecessary repetitive code.